infra/general/kEmails.php

```diff
@@ -76,16 +76,10 @@ protected static function getFormattedEmailComponentName($blockType, $roleName)
 		return $blockType . '-' . $roleName;
 	}
 	
-	public static function getDynamicTemplateBaseLink($roleName, $is2FA = false)
+	public static function getDynamicTemplateBaseLink($roleName, $baseLinkType)

```

Comment:
you can just put $baseLinkType = kEmails::DYNAMIC_EMAIL_BASE_LINK by default and send param only on the qr page case